x-pack/test/alerting_api_integration/security_and_spaces/config.ts

```diff
@@ -12,4 +12,6 @@ export default createTestConfig('security_and_spaces', {
   license: 'trial',
   ssl: true,
   enableActionsProxy: true,
+  // use hardcoded port if getPort doesn't return a free port
+  // proxyPort: 3232,
```

Comment:
Is there a way to have this configurable through the command line?

So something like this would work?

```
node scripts/functional_tests_server.js --config=test/alerting_api_integration/security_and_spaces/config.ts --xpack.actions.proxyUrl=http://localhost:3232
```

That might be better than uncommenting this line, because I think it's quite easy to uncomment this to run the test and then accidentally commit it in uncommented.
Might be better if we just encourage our dev to override the random port using a command line arg (if possible).

Reply:
Checking `process.env` for an override seems ok to me.  Would we even really need it though?

Reply:
The only way I found, to have this configurable through the command line, is to expose additional `userOption` in [args.js](https://github.com/elastic/kibana/blob/master/packages/kbn-test/src/functional_tests/cli/start_servers/args.js).